README.md

```diff
@@ -3,23 +3,19 @@
 <img align="left" src="https://avatars.githubusercontent.com/u/125200832?s=128&v=4" alt="ratatui logo">
 
 This is the documentation project for [Ratatui](https://github.com/tui-rs-revival/ratatui), a Rust
-library to build rich terminal user interfaces and dashboards. The documentation can be read at
-<https://tui-rs-revival.github.io/ratatui-book/>.
+library to build rich terminal user interfaces and dashboards. Note that this documentation is supplementary
+to the technical documentation, and we also maintain crate documentation. This is more of a guide book
+to help our users understand how to build a TUI with Ratatui. If you are looking for documentation on
+structs, enums, traits, methods, etc., please visit our [crates.io documentation](https://docs.rs/ratatui/0.21.0/ratatui/).
 
 [![License](https://img.shields.io/crates/l/ratatui?style=for-the-badge)](./LICENSE)
-[![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/tui-rs-revival/ratatui-book/mdbook.yml?label=Github%20Pages%20Deployment&logo=github&style=for-the-badge)](https://github.com/tui-rs-revival/ratatui-book/actions/workflows/mdbook.yml)
-[![Read the documentation](https://img.shields.io/badge/Read-Documentation-blue?style=for-the-badge)](https://tui-rs-revival.github.io/ratatui-book/)
 [![Discord](https://img.shields.io/discord/1070692720437383208?label=discord&logo=discord&style=for-the-badge)](https://discord.gg/pMCEU9hNEj)
 
-## Contributing
-
-The process for writing these docs is just getting started. Feel free to give some suggestions via
-[GitHub Discussions](https://github.com/tui-rs-revival/ratatui-book/discussions) or chat with us on
-[Discord](https://discord.com/channels/1070692720437383208/1115053951000268832).
+## Viewing the Book
 
-## Installation
+To read the book in your browser, click [here](https://tui-rs-revival.github.io/ratatui-book/).
```

Comment:
Can you phrase this to link some other word than “here”? even a bare url would be a improvement.

Reply:
Why is the link text an issue? Is this due to the fact that some people will be viewing this in a context where hyperlinks are not supported?